flexget/plugins/plugin_configure_series.py

```diff
@@ -17,6 +17,14 @@
 Base = db_schema.versioned_base('import_series', 0)
 
 
+def supported_ids():
+    # Return a list of supported series identifier as registered via their plugins
+    ids = []
+    for plugin_ in plugin.get_plugins(group='series_metainfo'):
```

Comment:
_ prepended variables suck, avoid if you can .. in this case single letter variable "p" is fine or something else